ee/clickhouse/migrations/0022_person_distinct_id2.py

```diff
@@ -0,0 +1,13 @@
+from infi.clickhouse_orm import migrations
+
+from ee.clickhouse.sql.person import (
+    KAFKA_PERSON_DISTINCT_ID2_TABLE_SQL,
+    PERSON_DISTINCT_ID2_MV_SQL,
+    PERSON_DISTINCT_ID2_TABLE_SQL,
+)
+
+operations = [
+    migrations.RunSQL(PERSON_DISTINCT_ID2_TABLE_SQL()),
```

Comment:
q: why are we making some of these functions and some not? (mind you I have not looked at `improve-clickhouse-schema-tests` yet)

Reply:
It's because of that branch - we are adding snapshot tests and otherwise we wouldn't be able to test how the queries look like with CLICKHOUSE_REPLICATED